config/manager/default_images.yaml

```
@@ -0,0 +1,33 @@
+---
+base: quay.io/piraeusdatastore
+components:
+  linstor-controller:
+    tag: v1.20.0
+    image: piraeus-server
+  linstor-satellite:
+    tag: v1.20.0
+    image: piraeus-server
+  linstor-csi:
+    tag: v0.21.0
+    image: piraeus-csi
+  drbd-reactor:
+    tag: v0.9.0
+    image: drbd-reactor
+  drbd-module-loader:
+    tag: v9.2.1
+    image: drbd9-jammy # Fall back image: chose a very recent-ish kernel, which hopefully can compile whatever config is actually in use
```

"very recent-ish" is a bit confusing. "Fallback" is one word. Maybe:

```suggestion
    image: drbd9-jammy # Fallback image: chose a fairly recent kernel, which can hopefully compile whatever config is actually in use
```